src/components/MainView/CreateDashboard/PushDashboardtoDb.js

```diff
@@ -0,0 +1,11 @@
+import db from '../../../fire';
```

Comment:
maybe it would be better to have something like 'utils' file and use it for entire project?